caffe2/proto/torch.proto

```diff
@@ -416,11 +466,36 @@ message TypeProto {
     optional TensorShapeProto shape = 2;
```

Comment:
We're currently storing strides in Dimension of TensorShapeProto. For example, if a tensor is of shape (2, 2) and strides (1, 2), we store 2, 2, 1, 2 in Dimension. Would be nice to store these separately.

Reply:
Good point, I will also add `stride` field into TensorShapeProto.